ut/performance_tests.cpp

```diff
@@ -80,7 +80,16 @@ class ColumnPerformanceTest : public ::testing::Test {
 
 TYPED_TEST_CASE_P(ColumnPerformanceTest);
 
+// Turns out this is the easiest way to skip test with current version of gtest
+#ifndef NDEBUG
+#  define SKIP_IN_DEBUG_BUILDS() do { std::cerr << "Test skipped...\n"; return; } while(0)
+#else
+#  define SKIP_IN_DEBUG_BUILDS() (void)(0)
+#endif
+
 TYPED_TEST_P(ColumnPerformanceTest, SaveAndLoad) {
+    SKIP_IN_DEBUG_BUILDS();
```

Comment:
FYI, Google Test has `GTEST_SKIP()` starting from some version, and, alternatively, you can prepend `DISABLED_` to the name of a test to disable it (should be available in the bundled `gtest` repo). Not an issue here, though.